C/services/south/ingest.cpp

```diff
@@ -28,6 +28,76 @@ static void ingestThread(Ingest *ingest)
 	}
 }
 
+/**
+ * Fetch all asset tracking tuples from DB and populate local cache
+ *
+ * @param m_mgtClient	Management client handle
+ */
+void Ingest::populateAssetTrackingCache(ManagementClient *m_mgtClient)
```

Comment:
m_ variables are by convention member variables rather than arguments. Although we don't always remember to make all member variables m_* and we probably shoal be better at this.

Reply:
Changed.